components/OutivityDetail/index.js

```diff
@@ -20,33 +27,47 @@ export default function OutivityDetail({ outivity, onDeleteOutivity }) {
       <Head>
         <title>{outivity.title}</title>
       </Head>
-      <main>
-        <article>
-          <ul>
-            <li>
-              <h2>{outivity.title}</h2>
-              <StyledOutivityImage
-                src={outivity.image}
-                alt={outivity.title}
-                width={300}
-                height={200}
-                layout="fixed"
-              />
-              <p>
-                <strong>Location: </strong> {outivity.area},
-                <StyledCountryName>{outivity.country}</StyledCountryName>
-              </p>
+      {!isEdit ? (
+        <main>
+          <article>
+            <ul>
+              <li>
+                <h2>{outivity.title}</h2>
+                <StyledOutivityImage
+                  src={outivity.image}
+                  alt={outivity.title}
+                  width={300}
+                  height={200}
+                  layout="fixed"
+                />
+                <p>
+                  <strong>Location: </strong> {outivity.area},
+                  <StyledCountryName>{outivity.country}</StyledCountryName>
+                </p>
 
-              <p>
-                <strong>Description: </strong> {outivity.description}
-              </p>
-            </li>
-          </ul>
-          <StyledDeleteButton type="button" onClick={confirmDelete}>
-            delete
-          </StyledDeleteButton>
-        </article>
-      </main>
+                <p>
+                  <strong>Description: </strong> {outivity.description}
+                </p>
+              </li>
+            </ul>
+            <StyledDeleteButton type="button" onClick={confirmDelete}>
+              delete
```

Comment:
```suggestion
              Delete
```
You might wanna change this so both your buttons start with a uppercase letter :)

Reply:
oopsie